packages/pwa-kit-dev/README.md

```diff
@@ -1 +1,23 @@
-# PWA Kit Developer Tools
+# PWA Kit Developer Tools 🛠️
```

Comment:
```suggestion
# PWA Kit Developer Tools
```

For consistency with other README files in our repo, use emoji only on h2s (and put the emoji before the heading text, not after).